index.html

```diff
@@ -178,13 +178,14 @@ <h3 class="work-titl work-titl-form">CONTACT US</h3>
                 needs coding. I’d love to help with it
             </p>
         </div>
-        <form action="https://formspree.io/f/xlevdoyw" method="POST">
+        <form id="form" action="https://formspree.io/f/xlevdoyw" method="POST">
             <ul>
                 <li>
                     <input type="text" name="user_name" id="user_name" placeholder="Your Name" minlength="5"
                         maxlength="30" required>
                 </li>
                 <li><input type="email" name="user_mail" id="user_mail" placeholder="jeremy@gmail.com" required></li>
+                <li class="msg">Message</li>
                 <li>
                     <textarea name="user_text" id="user_text" cols="46" rows="10" placeholder="Enter your message here"
                         minlength="10" maxlength="500" required></textarea>
```

Comment:
- [ ] Please kindly ensure that the error message appears close to the button as per requirements.